### PR TITLE
Update kind setup

### PR DESCRIPTION
**Pull request by graphaelli**

### CODE REVIEW

File: hack/kind/kind.sh [hunk 1]
```diff
@@ -86,17 +86,18 @@ function setup_kind_cluster() {
...
   # Create Kind cluster
-  if ! (kind create cluster --name=${CLUSTER_NAME} ${config_opts} --loglevel "${KIND_LOG_LEVEL}" --retain --image "${NODE_IMAGE}"); then
+  if ! (kind ${KIND_LOGGING} create cluster --name=${CLUSTER_NAME} ${config_opts} --retain --image "${NODE_IMAGE}"); then
     echo "Could not setup Kind environment. Something wrong with Kind setup."
     exit 1
   fi
 
-  KUBECONFIG="$(kind get kubeconfig-path --name="${CLUSTER_NAME}")"
-  export KUBECONFIG
+  # persist kubeconfig for reliabililty in following kubectl commands
```
Comment (anyasabo) [inline hack/kind/kind.sh]: does the path change or what is the issue with the original variable? it's not clear to me what is unreliable about it

Comment (graphaelli) [reply hack/kind/kind.sh]: good point. `kind` sets the default context when creating a cluster so if not running kind elswhere in parallel, these `kubectl` commands would just work without this `kubeconfig`.  If that's not something to worry about we can remove this altogether.

Comment (anyasabo) [reply hack/kind/kind.sh]: ah got it thank you. I'm not sure when we'll be doing that but I haven't worked with it much so it's definitely possible I'm missing something obvious.

**Comment by graphaelli** [comment]
Thanks for reviewing - just a reminder that I'm not authorized to merge so please go ahead if you're ok with these changes.

**Pull request merged**